Mathlib/Algebra/Order/Module/Defs.lean

```diff
@@ -822,6 +803,232 @@ instance instSMulPosReflectLE [SMulPosReflectLE α β] : SMulPosReflectLE α β
 end Right
 end OrderDual
 
+section OrderedRing
+variable [OrderedRing α]
+
+section OrderedAddCommGroup
+variable [OrderedAddCommGroup β] [Module α β]
+
+#noalign eq_of_smul_eq_smul_of_neg_of_le
+
+section PosSMulMono
+variable [PosSMulMono α β]
+
+lemma smul_le_smul_of_nonpos (h : b₁ ≤ b₂) (ha : a ≤ 0) : a • b₂ ≤ a • b₁ := by
+  rw [← neg_neg a, neg_smul, neg_smul (-a), neg_le_neg_iff]
+  exact smul_le_smul_of_nonneg_left h (neg_nonneg_of_nonpos ha)
+#align smul_le_smul_of_nonpos smul_le_smul_of_nonpos
+
+lemma antitone_smul_left (ha : a ≤ 0) : Antitone (SMul.smul a : β → β) :=
+  fun _ _ h => smul_le_smul_of_nonpos h ha
+#align antitone_smul_left antitone_smul_left
+
+instance PosSMulMono.toSMulPosMono : SMulPosMono α β where
+  elim _b hb a₁ a₂ ha := by rw [← sub_nonneg, ← sub_smul]; exact smul_nonneg (sub_nonneg.2 ha) hb
+
+end PosSMulMono
+
+section PosSMulStrictMono
+variable [PosSMulStrictMono α β]
+
+lemma smul_lt_smul_of_neg (hb : b₁ < b₂) (ha : a < 0) : a • b₂ < a • b₁ := by
+  rw [← neg_neg a, neg_smul, neg_smul (-a), neg_lt_neg_iff]
+  exact smul_lt_smul_of_pos_left hb (neg_pos_of_neg ha)
+#align smul_lt_smul_of_neg smul_lt_smul_of_neg
+
+lemma strict_anti_smul_left (ha : a < 0) : StrictAnti (SMul.smul a : β → β) :=
```

Comment:
```suggestion
lemma strictAnti_smul_left (ha : a < 0) : StrictAnti (SMul.smul a : β → β) :=
```

Reply:
This is on my TODO list, but I'm not renaming anything in this PR!